package.json

```diff
@@ -53,15 +53,15 @@
     "precommit-test"
   ],
   "devDependencies": {
+    "@appium/eslint-config-appium": "^6.0.2",
+    "@appium/gulp-plugins": "^7.0.2",
```

Comment:
it would be cool to eventually get rid of `@appium/gulp-plugins` dependenncy at all and have all scripts executed without the outdated/unmaintained gulp thing like it was done for the main monorepo.

Reply:
yeah, I want this to happen too.  we'll get to it eventually